servant-jquery/src/Servant/JQuery/Internal.hs

```diff
@@ -25,8 +25,29 @@ import GHC.Exts (Constraint)
 import GHC.TypeLits
 import Servant.API
 
+data CommonGeneratorOptions = CommonGeneratorOptions {
+        -- Function transforming function names
+        functionName :: String -> String,
```

Comment:
`functionName` doesn't seem like the right name for this field. `functionNameModifier`, `functionRenamer` or some such?